lms/djangoapps/ccx/views.py

```diff
@@ -179,6 +179,13 @@ def dashboard(request, course, ccx=None):
                 model='CourseEnrollment',
             )
         )
+        # if course licensing is enabled, pending enrollments will be in the context.
+        context['pending_ccx_members'] = (
+            run_extension_point(
+                'PCO_GET_PENDING_ENROLLMENTS_FOR_CCX',
+                ccx_id=ccx_locator,
+            )
+        )
```

Comment:
I know you are validating whether course licensing is enable. But what do you think about checking that first before calling a run_extension_point. @anfbermudezme 

From the perspective of the platform, I can not anticipate the response.
```suggestion
        # if course licensing is enabled, pending enrollments will be in the context.
        context['pending_ccx_members'] = run_extension_point(
                'PCO_GET_PENDING_ENROLLMENTS_FOR_CCX',
                ccx_id=ccx_locator,
            ) if if run_extension_point('PCO_ENABLE_COURSE_LICENSING') else ()
```

Reply:
`'ENABLE_COURSE_LICENSING'` validation should be through the plugin, not on the platform precisely to avoid these double validations. It may be the case where 'PCO_ENABLE_COURSE_LICENSING' works and 'PCO_GET_PENDING_ENROLLMENTS_FOR_CCX' does work, reproducing the error I show at https://github.com/Pearson-Advance/openedx-themes/pull/224

Reply:
The reason of the error in themes is precisely the fact you can not anticipate* the result by just executing an run_extension_point, with my suggestion there is no error. However this is not the point.

My concerns are:
1. I don't think we need to move the validations to the plugin as we already have one `run_extension_point('PCO_ENABLE_COURSE_LICENSING')`.
2. Moving the validation to the plugin implies that for every old and upcoming run_extension_point/function-in plugin side we are checking if the plugin is enabled. But those functions might be reusable in the future by other parts within the plugin and checking if plugin is enabled inside the plugin makes no sense.

I don't see a double validation. I see a validation and an action. It is only a double validation if you check the plugin is enable inside the run_extension_point.

*by anticipating I mean that your current implementation return iterable and a non iterable variable.
1. Plugin is not installed. (None is returned)
2. Couse Licensing is False (not enable). (None is returned)
3. Course Licensing is True (queryset which is an iterable)


@anfbermudezme